boltzrpc/boltzrpc.proto

```diff
@@ -418,17 +419,44 @@ enum IncludeSwaps {
     AUTO = 2;
 }
 
+message AnySwapInfo {
+    string id = 1;
+    SwapType type = 2;
+    Pair pair = 3;
+    SwapState state = 4;
+    optional string error = 5;
+    string status = 6;
+    // the "from" amount of the swap.
+    // its the expected amount sent to be to the lockup address for submarine and chain swaps and
+    // the invoice amount for reverse swaps.
+    // the "to" amount can be calculated by subtracting the service and network fee.
+    uint64 amount = 7;
+    int64 created_at = 8;
+    optional uint64 service_fee = 9;
+    // inclues the routing fee for reverse swaps
+    optional uint64 onchain_fee = 10;
+    bool is_auto = 11;
+    uint64 tenant_id = 12;
+}
+
 message ListSwapsRequest {
     optional Currency from = 1;
     optional Currency to = 2;
     optional SwapState state = 4;
     IncludeSwaps include = 5;
+    optional uint64 limit = 6;
+    optional uint64 offset = 7;
+    // wether to return swaps in the shared `all_swaps` list or in the detailed lists.
+    // the `limit` and `offset` are only considered when `unify` is true.
```

Comment:
Wouldn't throwing an error be nicer when trying to paginate a non-unified call? Silently ignoring sounds like a bad idea